hack/qemu-metadata-api/main.go

```diff
@@ -13,29 +13,38 @@ import (
 	"log/slog"
 	"os"
 
+	"github.com/edgelesssys/constellation/v2/hack/qemu-metadata-api/dhcp/dnsmasq"
+	"github.com/edgelesssys/constellation/v2/hack/qemu-metadata-api/dhcp/virtwrapper"
 	"github.com/edgelesssys/constellation/v2/hack/qemu-metadata-api/server"
-	"github.com/edgelesssys/constellation/v2/hack/qemu-metadata-api/virtwrapper"
 	"github.com/edgelesssys/constellation/v2/internal/logger"
 	"libvirt.org/go/libvirt"
 )
 
 func main() {
 	bindPort := flag.String("port", "8080", "Port to bind to")
-	targetNetwork := flag.String("network", "constellation-network", "Name of the network in QEMU to use")
+	targetNetwork := flag.String("network", "constellation-network", "Name of the network in libvirt")
 	libvirtURI := flag.String("libvirt-uri", "qemu:///system", "URI of the libvirt connection")
+	leasesFileName := flag.String("dnsmasq-leases", "./dnsmasq.leases", "Path to the dnsmasq leases file")
```

Comment:
```suggestion
	leasesFileName := flag.String("dnsmasq-leases", "", "Path to the dnsmasq leases file")
```
The default behavior should probably stay libvirt, so we should not set a default value for `dnsmasq-leases`, otherwise one needs to explicitly unset the flag to use the metadata api with libvirt.